hepcrawl/parsers/jats.py

```diff
@@ -314,10 +314,25 @@ def get_affiliation(self, id_):
                 no match.
         """
         affiliation_node = self.root.xpath('//aff[@id=$id_]', id_=id_)[0]
```

Comment:
I would make the selector stricter here instead having the LiteratureBuilder clean up the empty affiliation later: something like `//aff[@id=$id_ and not(./email)]` should do the job.